docs/how-to-guides/new-user-guides/kubernetes-resources-setup/load-balancer-and-ingress-controller/ingress-configuration.md

```diff
@@ -15,9 +15,9 @@ Because of this change to the controller, the default behavior no longer sets `h
 
 If you use this option, Ingress routes requests for a hostname to the service or workload that you specify.
 
-1. Enter the **Request Host** that your Ingress controller will handle request forwarding for. For example, `www.mysite.com`.
+1. Specify a path of type `Prefix` and a specify a path such as `/`.
```

Comment:
```suggestion
1. Specify a path of type `Prefix` and specify a path such as `/`.
```

Reply:
Fixed in https://github.com/rancher/rancher-docs/pull/557/commits/13e4b8e6f15e08c83dc815fae2767d331c573593.